students/andrew_garcia/lesson_08/Circle.py

```diff
@@ -0,0 +1,64 @@
+'''
+Andrew Garcia
+Circle
+8/19/19
+'''
+
+import math
+
+class Circle(object):
+    def __init__(self, radius):
+        self.radius = radius
+
+    @property
+    def diameter(self):
+        return self.radius * 2
+
+    @diameter.setter
+    def diameter(self, diameter):
+        self.radius = diameter/2
+
+    @property
+    def area(self):
+        return round(math.pi * (self.radius ** 2), 3)
+
+    @classmethod
+    def from_diameter(cls, diameter):
+        cls.radius = diameter/2
```

Comment:
this is incorrect, should be `return cls(diameter / 2)`

this is typically used as an alternate constructor like:
```
circle = Circle.from_diameter(10)
```

the way you currently have it will not satisfy above use case.